f8a_worker/workers/downstream.py

```diff
@@ -1,253 +0,0 @@
-        self._strict_assert(arguments.get('name'))
-        self._strict_assert(arguments.get('version'))
-
-        eco = arguments['ecosystem']
-        pkg = arguments['name']
-        tool_responses = {}
-        result_summary = {
-            'package_names': [],
-            'registered_srpms': [],
-            'all_rhn_channels': [],
-            'all_rhsm_content_sets': [],
-            'all_rhsm_product_names': []
-        }
-        result_data = {'status': 'error',
-                       'summary': result_summary,
-                       'details': tool_responses
-                       }
-
-        # bail out early; we need access to internal services or the package is
-        # from Maven ecosystem, otherwise we can't comment on downstream usage
-        is_maven = Ecosystem.by_name(self.storage.session, eco).is_backed_by(EcosystemBackend.maven)
-        if not self._is_inside_rh() and not is_maven:
-            return result_data
-
-        self.log.debug('Fetching {e}/{p} from Anitya'.format(e=eco, p=pkg))
-        res = self._fetch_anitya_project(eco, pkg)
-        anitya_rpm_names = []
-        anitya_mvn_names = []
-        if res is None:
-            result_data['status'] = 'error'
-        elif res.status_code == 200:
-            self.log.debug('Retrieved {e}/{p} from Anitya'.format(e=eco, p=pkg))
-            anitya_response = res.json()
-            tool_responses['redhat_anitya'] = anitya_response
-            # For now, we assume all downstreams are ones we care about
-            for entry in anitya_response['packages']:
-                if entry['distro'] == RH_RPM_DISTRO_NAME:
-                    anitya_rpm_names.append(entry['package_name'])
-                elif entry['distro'] == RH_MVN_DISTRO_NAME:
-                    anitya_mvn_names.append(entry['package_name'])
-                else:
-                    self.log.warning(
-                        'Unknown distro {d} for downstream package {o} (package {p}) in Anitya'.
-                        format(d=entry['distro'], o=entry['package_name'], p=pkg)
-                    )
-            self.log.debug('Candidate RPM names from Anitya: {}'.format(anitya_rpm_names))
-            self.log.debug('Candidate MVN names from Anitya: {}'.format(anitya_mvn_names))
-            # TODO: Report 'partial' here and switch to 'success' at the end
-            result_data['status'] = 'success'
-        else:
-            msg = 'Failed to find Anitya project {e}/{p}. Anitya response: {r}'
-            self.log.error(msg.format(e=eco, p=pkg, r=res.text))
-            result_data['status'] = 'error'
-
-        if self._is_inside_rh():
-            # we have candidate downstream name mappings, check them against Brew
-            seed_names = anitya_rpm_names or [self._prefix_package_name(pkg, eco)]
-            self.log.debug('Checking candidate names in Brew: {}'.format(seed_names))
-
-            args = ['brew-utils-cli', '--version', arguments['version']]
```

Comment:
We don't need to [install brew-utils in worker-base image](https://github.com/fabric8-analytics/fabric8-analytics-worker-base/blob/master/hack/install_deps_rpm.sh#L33) then.